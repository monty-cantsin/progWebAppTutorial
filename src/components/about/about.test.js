import React from 'react'
import ReactDOM from 'react-dom'
import ReactTestRender from 'react-test-renderer'
import toJson from 'enzyme-to-json'
import { shallow } from 'enzyme'

import About from './about'

test('About component should render as expected', () => {
	const component = shallow(<About />)
	const tree = toJson(component)

	expect(tree).toMatchSnapshot()
})