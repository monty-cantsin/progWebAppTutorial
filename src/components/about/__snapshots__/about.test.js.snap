// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`About component should render as expected 1`] = `
<div>
  <h1
    className="heading"
  >
    Hi, I'm Taylor!
  </h1>
  <p
    className="bio"
  >
     This section is all about me!
  </p>
</div>
`;
